cluster-autoscaler/simulator/drain_test.go

```diff
@@ -49,15 +48,6 @@ func TestGetPodsToMove(t *testing.T) {
 			OwnerReferences: GenerateOwnerReferences("rs", "ReplicaSet", "extensions/v1beta1", ""),
 		},
 	}
-	manifestPod := &apiv1.Pod{
```

Comment:
Why remove the `manifestPod` test cases? They should be still passing and they indirectly test the default rules set - seems useful to keep them.

Reply:
Ok, let me bring that back.